src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -4853,8 +4883,8 @@ def broken_func():
     with pytest.raises(GenerationError) as err:
         loop.load(kernel)
     assert ("Generation Error: Unexpected function space found. Expecting "
-            "one of ['w3', 'wtheta', 'w2v', 'w0', 'w1', 'w2', 'w2h', "
-            "'any_w2'] but found 'broken'" in str(err))
+            "one of ['w3', 'wtheta', 'w2v', 'w2broken', 'w0', 'w1', 'w2', "
```

Comment:
Please use the actual list when comparing.

Reply:
Done, thanks for the suggestion.